tools/c7n_azure/pyproject.toml

```diff
@@ -40,7 +40,7 @@ azure-mgmt-datafactory = "^1.0.0"
 azure-mgmt-desktopvirtualization = "^1.0.0"
 azure-mgmt-dns = "8.0.0b1"
 azure-mgmt-eventgrid = "^8.0.0"
-azure-mgmt-eventhub = "^8.0.0"
+azure-mgmt-eventhub = "11.0.0"
```

Comment:
version markers shouldn't be hard pins ideally, they should be carrot ideally re ^

Reply:
so that we can get updates, for releases we freeze the dependency graph in the wheels.

Reply:
Ok, i added a caret symbol